pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -43,9 +51,17 @@ const (
 	minRetryDelay = 2 * time.Second
 	maxRetryDelay = 120 * time.Second
 
-	// Default number of workers processing a resource change
-	defaultWorkers = 1
-	workerItemKey  = "key"
+	workerItemKey = "key"
+
+	TrafficEncryptionModeWireGuard = "wireGuard"
```

Comment:
Could not we reuse `config.TrafficEncryptionModeWireGuard`?

Reply:
Yes, using `config.TrafficEncryptionModeWireGuard` now.